### PR TITLE
Add googleapis and grpc-java

### PR DESCRIPTION
**Pull request by keith**
These are in the same PR since they have a circular dependency on each other

Fixes: https://github.com/bazelbuild/bazel-central-registry/issues/353
Fixes: https://github.com/bazelbuild/bazel-central-registry/issues/1113

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (googleapis, grpc-java) have been updated in this PR. Please review the changes.

**Review by meteorcloudy — APPROVED** [review]
This is great! Although I'd like to see the patches upstreamed, but I'm fine with merging this one before that to unblock other progresses! Thanks a lot!

**Comment by keith** [comment]
upstream prs are up:

- https://github.com/grpc/grpc-java/pull/11046
- https://github.com/googleapis/googleapis/pull/892

but thanks for reviewing, either way we need this one first because of the circular dependency

**Pull request merged**
